content/docs/cli/_index.md

```diff
@@ -0,0 +1,523 @@
+---
+title: "CLI"
+linkTitle: "CLI"
+weight: 9
+Description: >
+  CLI for Dell Container Storage Modules (CSM)
+---
+dellctl is a common command line interface(CLI) used to interact with and manage your [Container Storage Modules](https://github.com/dell/csm) (CSM) resources.
+This document outlines all dellctl commands, their intended use, options that can be provided to alter their execution, and expected output from those commands.
+
+| Command | Description |
+| - | - |
+| [dellctl](#dellctl) | dellctl is used to interact with Container Storage Modules |
+| [dellctl cluster](#dellctl-cluster) | Manipulate one or more k8s cluster configurations |
+| [dellctl cluster add](#dellctl-cluster-add) | Add a k8s cluster to be managed by dellctl |
+| [dellctl cluster remove](#dellctl-cluster-remove) | Removes a k8s cluster managed by dellctl |
+| [dellctl cluster get](#dellctl-cluster-get) | List all clusters currently being managed by dellctl |
+| [dellctl backup](#dellctl-backup) | Allows to manipulate application backups/clones |
+| [dellctl backup create](#dellctl-backup-create) | Create an application backup/clones |
+| [dellctl backup delete](#dellctl-backup-delete) | Delete application backups |
+| [dellctl backup get](#dellctl-backup-get) | Get application backups |
+| [dellctl restore](#dellctl-restore) | Allows to manipulate application restores |
+| [dellctl restore create](#dellctl-restore-create) | Restore an application backup |
+| [dellctl restore delete](#dellctl-restore-delete) | Delete application restores |
+| [dellctl restore get](#dellctl-restore-get) | Get application restores |
+
+
+## Installation instructions
+1. Download `dellctl` from [here](https://github.com/dell/csm/releases/tag/v1.4.0)
+2. Chmod +x dellctl
+3. Move `dellctl` to `/usr/loca/bin` or add `dellctl`'s containing directory path to PATH environment variable.
```

Comment:
Should `/usr/loca/bin` be `/usr/local/bin`

Reply:
fixed the typo. Thanks for pointing it out!